config/swarm-sip.yml

```diff
@@ -1,14 +1,9 @@
 ---
-# SIP network interface names
-ext_name: ilab
-bdn_name: p3-bdn
-lln_name: p3-lln
-
 # SIP configuration
 cluster_user: "fedora"
 cluster_state: "present"
 cluster_name: "swarm-sip"
-cluster_template_name: "swarm-fedora-atomic-29"
```

Comment:
What's the reason for the downgrade here?

Reply:
`swarm-fedora-atomic-29` doesn't have Docker CE sadly.

Reply:
Didn't you manually install Docker CE in the Fedora image previously?

Reply:
I did, it’s probably better to automate it as an Ansible playbook, rather than the manual configuration of the image, so let’s use fa27 until then.

Reply:
Would have preferred to see this kind of change separated from the refactor.